Commands/ApiEndpointCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace ShopwarePlugins\Connect\Commands;
+
+use Shopware\Commands\ShopwareCommand;
+use Shopware\Components\ConfigWriter;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class ApiEndpointCommand extends ShopwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('connect:endpoint:set')
+            ->setDescription('Set the API endpoint to a custom location')
+            ->addArgument(
+                'api-endpoint',
+                InputArgument::REQUIRED,
+                'URL to of the API endpoint.'
+            )
+            ->addArgument(
```

Comment:
Do we need shopId? Currently different subshops can't work separately with different environments of Connect/SEM